components/TimeSeries.vue

```diff
@@ -0,0 +1,323 @@
+    });
+  });
+
+  return lines;
+});
+
+const interventionsPlotBands = computed(() => {
+  const bands = Array<Highcharts.AxisPlotBandsOptions>();
+
+  if (!usePlotBands) {
+    return bands;
+  }
+
+  appStore.interventionsData?.forEach(({ id, start, end }) => {
+    const interventionLabel = appStore.metadata?.results.interventions.find(({ id: interventionId }) => interventionId === id)?.label;
+    const text = `${interventionLabel ? `${interventionLabel} : ` : ""}Days ${start} to ${end}`;
+    bands.push({ from: start, to: end, color: plotBandsColor, label: { text } });
+  });
+
+  return bands;
+});
+
+const handleAccordionToggle = () => {
+  emit("toggleOpen");
+};
+
+const onMove = () => {
+  emit("syncTooltipsAndCrosshairs");
+};
+
+const handleMouseLeave = () => {
+  emit("hideAllTooltips");
+};
+
+const handleMouseOver = () => {
+  emit("showAllTooltips");
+};
+
+// Override the reset function as per synchronisation demo: https://www.highcharts.com/demo/highcharts/synchronized-charts
+Highcharts.Pointer.prototype.reset = () => {
+  return undefined;
+};
+
+const setChartHeight = debounce(async (height: number) => {
+  chart.setSize(undefined, height, { duration: 250 });
+}, 10);
+
+// Resize the chart when the accordion is opened or closed.
+watch(() => [props.chartHeightPx, props.open], () => {
+  setChartHeight((props.open ? props.chartHeightPx : props.minChartHeightPx));
+});
+
+const chartInitialOptions = () => {
+  return {
+    chart: {
+      height: props.chartHeightPx,
+      marginLeft: 75, // Specify the margin of the y-axis so that all charts' left edges are lined up
+      backgroundColor: chartBackgroundColor,
+      events: {
+        fullscreenOpen() {
+          this.update({ chart: { backgroundColor: chartBackgroundColorOnExporting } });
+        },
+        fullscreenClose() {
+          this.update({ chart: { backgroundColor: chartBackgroundColor } });
+        },
+      },
+      style: {
+        fontFamily: "ImperialSansText, sans-serif", // TODO: Make the font-family derive from a globally configurable constant
+      },
+    },
+    exporting: {
+      filename: `${seriesMetadata.value!.label} in ${appStore.currentScenario.parameters?.country}`,
+      chartOptions: {
+        title: {
+          text: seriesMetadata.value!.label,
+        },
+        subtitle: {
+          text: seriesMetadata.value!.description,
+        },
+        chart: {
+          backgroundColor: chartBackgroundColorOnExporting,
+          height: 500,
+        },
+      },
+      buttons: {
+        contextButton: {
+          height: 20,
+          width: 22,
+          symbolSize: 12,
+          symbolY: 10,
+          symbolX: 12,
+          symbolStrokeWidth: 2,
+          // Omit 'printChart' and 'viewData' from menu items
+          menuItems: ["printChart", "downloadCSV", "downloadXLS", "separator", "downloadPNG", "downloadJPEG", "downloadPDF", "downloadSVG", "separator", "viewFullscreen"],
```

Comment:
Ah, you've added printChart back in - I think it's fine to leave it out as you said, if you get unwanted tooltip. Either way, the comment above should be consistent with the list.

Reply:
Oh, that was an accident.